docs/explanation/charm-architecture.md

```diff
@@ -4,50 +4,24 @@ A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) t
 
 Conceptually, the charm can be divided into the following:
 
-- Management of LXD ephemeral virtual machines to host [ephemeral self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/autoscaling-with-self-hosted-runners#using-ephemeral-runners-for-autoscaling)
+- Management of Openstack virtual machines to host self-hosted runners
 - Management of the virtual machine image
 - Management of the network
 - GitHub API usage
 - Management of [Python web service for checking GitHub repository settings](https://github.com/canonical/repo-policy-compliance)
 - Management of dependencies
 
-## LXD ephemeral virtual machines
+## Ephemeral virtual machines
 
-To ensure a clean and isolated environment for every runner, self-hosted runners use LXD virtual machines. The charm spawns virtual machines, setting resources based on charm configurations. The self-hosted runners start with the ephemeral option and will clean themselves up once the execution has finished, freeing the resources. This is [similar to how GitHub hosts their runners due to security concerns](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#self-hosted-runner-security).
+To ensure a clean and isolated environment for every runner, self-hosted runners use Openstack virtual machines. The charm spawns virtual machines, setting resources based on charm configurations. The self-hosted runners start with the ephemeral option and will clean themselves up once the execution has finished, freeing the resources. This is [similar to how GitHub hosts their runners due to security concerns](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#self-hosted-runner-security).
```

Comment:
I think the part for ephemeral is not really true, as it only applied to LXD. The VM's have to be cleaned up upon next reconciliation. Maybe delete the phrase for now.

Reply:
Thanks! rephrased a bit to remove the `ephemeral` part.

Reply:
yeah they are ephemeral in the sense that they unregister from github after the run, but the "clean themselves up" part is no longer existing.